packages/perseus/src/widgets/label-image/label-image.tsx

```diff
@@ -64,15 +64,22 @@ type Point = {
     y: number;
 };
 
-type LabelImageProps = ChangeableProps &
-    DependencyProps &
-    Omit<PerseusLabelImageWidgetOptions, "markers"> & {
-        apiOptions: APIOptions;
-        // The list of label markers on the question image.
-        markers: ReadonlyArray<InteractiveMarkerType>;
-        // Whether the question has been answered by the user.
-        questionCompleted: boolean;
+export type OptionalAnswersMarkerType = Omit<
+    InteractiveMarkerType,
+    "answers"
+> & {
+    answers?: string[];
+};
+
+type Options = Omit<PerseusLabelImageWidgetOptions, "markers"> & {
+    // The list of label markers on the question image.
+    markers: ReadonlyArray<OptionalAnswersMarkerType>;
+};
+
+type Props = WidgetProps<Options> &
+    DependencyProps & {
         // preferred placement for popover (preference, not MUST)
+        // TODO: this is sus, probably never passed in
```

Comment:
Did a short search in Webapp and found this included in a label image schema in the widget.go file. Curious as it's not in `PerseusLabelImageWidgetOptions`.